compiler/arm/codegen/OMRTreeEvaluator.cpp

```diff
@@ -925,6 +926,184 @@ static void constLengthArrayCopyEvaluator(TR::Node *node, int32_t byteLen, TR::R
 
    }
 
+#ifdef J9_PROJECT_SPECIFIC
```

Comment:
As far as I know, arraycopy is supported at the OMR level. Could this guard be removed?

Reply:
Existing wrtbar-related functions are guarded by J9_PROJECT_SPECIFIC in OMRTreeEvaluator.cpp, so I did in the same way.  In addition, the new functions depend on the header file "j9modron.h".

Reply:
Hmm... I see.